src/proposals/20250109/SparkEthereum_20250109.t.sol

```diff
@@ -0,0 +1,926 @@
+        });
+        MarketParams memory ptsusdeoct =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_SUSDE_24OCT2024,
+            oracle:          PT_SUSDE_24OCT2024_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.86e18
+        });
+        MarketParams memory ptsusdedec =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_SUSDE_26DEC2024,
+            oracle:          PT_SUSDE_26DEC2024_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+        MarketParams memory ptsusdemar =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_SUSDE_27MAR2025,
+            oracle:          PT_SUSDE_27MAR2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+        MarketParams memory ptsusdemay =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_SUSDE_29MAY2025,
+            oracle:          PT_SUSDE_29MAY2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, usde945,    10_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, usde77,     1_000_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, susde945,   10_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, susde77,    1_000_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdeoct, 100_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdedec, 250_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemar, 400_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemay, 0);
+
+        executeAllPayloadsAndBridges();
+
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, usde945,    0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, usde77,     0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, susde945,   0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, susde77,    0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdeoct, 0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdedec, 0);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemar, 400_000_000e18, 500_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemay, 0,              200_000_000e18);
+
+        skip(1 days);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(ptsusdemar);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(ptsusdemay);
+        
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemar, 500_000_000e18);
+        _assertMorphoCap(Ethereum.MORPHO_VAULT_DAI_1, ptsusdemay, 200_000_000e18);
+    }
+
```

Comment:
Should add a test to ensure that CCTP works as intended with new mint recipient for both base and ethereum almProxys, actually would be best to copy the e2e test of mint, swap, bridge, bridge, swap, burn

Reply:
It probably makes sense to always have an e2e test here like we do with SparkLend.

Reply:
We can figure out how best to generalize these tests, but for now I will just add a one-off to test CCTP config in both directions.

Reply:
Added